### PR TITLE
change another way to render math

### PR DESCRIPTION
**Pull request by Jocs**
| Q                | A
| ---------------- | ---
| Bug fix?         | no
| New feature?     | yes
| BC breaks?       | no
| Deprecations?    | no
| New tests added? | not needed
| Fixed tickets    | #242 
| License          | MIT

### Description

[Description of the bug or feature]

### how to insert math block?

step1. input `$$`
step2. press enter

or

press `Option + Ctrl + M`

* [x] change to another way to preview math.
* [x] open markdown file with multiple mathematical formula.
* [x] save markdown file with multiple mathematical formula.
* [x] edit markdown file with multiple mathematical formula.
* [x] preview mathematical formula under the input area.
* [x] add multiple lines math \ display math \ inline math to menus.
* [x] handle dark theme

### CODE REVIEW

File: src/editor/contentState/updateCtrl.js [hunk 1]
```diff
@@ -284,6 +284,13 @@ const updateCtrl = ContentState => {
     return null
   }
 
+  ContentState.prototype.undateMathContent = function (block) {
```
Comment (fxha) [inline src/editor/contentState/updateCtrl.js]: typo?

Comment (Jocs) [reply src/editor/contentState/updateCtrl.js]: 😸 yes, my fault!

**Comment by Jocs** [comment]
@fxha I can not reproduce the problem 1.
can you show a gif image to me. thanks.

**Comment by Jocs** [comment]
@fxha I update most of the dependencies. So you need to reinstall all the dependencies after this PR merged!

>rm -rf node_modules
>npm i

**Comment by fxha** [comment]
>  I can not reproduce the problem 1.
> can you show a gif image to me. thanks.

I can't reproduce it too :/

But following text is displayed strange if you switch from source code mode to preview mode.

```
$$
\frac{3}{4}
\times
\frac{5}{2}
=6\frac{1}{2}
$$
```

**Problem 1:**

mt_bug_1

**Problem 2:**

Set cursor at the end of `$$`, switch to preview mode and press `enter`.

mt_bug_2

**Comment by Jocs** [comment]
fix problem 2 is another `cursor` bug when change mode from source code mode.

**Comment by Jocs** [comment]
fixed `TypeError: Cannot set property 'math' of null`

**Pull request merged**